app/assets/stylesheets/utilities/_typography.scss

```diff
@@ -99,3 +99,8 @@ h6,
 .h6 {
   @extend %h6;
 }
+
+h2.text-center--line {
```

Comment:
do we need to scope this to `h2`? or can it be a more generic style?

Reply:
(looks like this was already done!)

Reply:
yeah I looked at it again and removed it